modules/cluster-logging-about-logstore.adoc

```diff
@@ -2,6 +2,8 @@
 //
 // * logging/cluster-logging.adoc
 
+//Eng-Feedback-Req: Log store is Fluentd & Elasticsearch heavy - can these be replaced with generic terms?
+
 :_content-type: CONCEPT
 [id="cluster-logging-about-logstore_{context}"]
 = About the log store
```

Comment:
"Optionally, you can use the log forwarding features to forward logs to external log stores using Fluentd protocols, syslog protocols, or the {product-title} Log Forwarding API." ===>
"Optionally you can use the Log Forwarder API to forward logs to an external store. Several types of store are supported, including fluentd, rsyslog, kafka and others.